cluster/saltbase/salt/flannel-server/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Network": "192.168.0.0/16",
```

Comment:
why 192 (with /26 limited to 1024 nodes with no room ever to move) as opposed to 10.248.0.0/13 or something?